src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import 'vars';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &__day {
+    width: $day-width;
+    height: $day-width;
+    background-color: $calendar-color;
+    color: black;
+    border: $day-border;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $pointer-calendar-color;
+      transform: translateY(-20px);
+      transition-duration: 500ms;
+    }
```

Comment:
The `transition-duration` property should be applied to the element itself, not on the `:hover` pseudo-class. This ensures that the transition is smooth both when the mouse hovers over the element and when it leaves.